src/presenters/gladLPresenter.ts

```diff
@@ -125,8 +140,19 @@ class GLADLPresenter extends PresenterInterface<GladLAlertResultType, GladLPrese
     }
 
     async getDownloadURLs(startDate: string, endDate: string, params: Record<string, any>): Promise<{ csv: string, json: string }> {
-        const geostoreId: string = await GeostoreService.getGeostoreIdFromSubscriptionParams(params);
-        const uri: string = GLADLPresenter.#getURLForDownload(startDate, endDate, geostoreId);
+        const area: Record<string, any> = params.area ? await AreaService.getUserArea(params.area) : {};
+        const iso: Record<string, any> = area?.admin && Object.keys(area.admin).length 
```

Comment:
Same question as earlier
```suggestion
        const iso: Record<string, any> = area?.attributes?.admin && Object.keys(area.attributes.admin).length 
```